### DISCUSSION TITLE
Can not use 'Controller' with 'control' in custom component library from the version above 7.43.2

### DISCUSSION THREAD

**Discussion by ikbal-nayem** [react-hook-form/react-hook-form#10194] [General]
I have created a component library with the version `7.43.2`, where I used "**Controler**" with "**control**" for some component to handle user input. And it was working fine.
Whenever I tried to update the version from `7.43.2` to `7.43.3` or above I am getting the following error in my console,

![image](https://user-images.githubusercontent.com/31121811/228280599-ba063b92-f827-4ba8-aab9-59ed30f5390b.png)

### In my component library,
![image](https://user-images.githubusercontent.com/31121811/228289813-bd5f5efd-9f2c-46b0-a7d2-4fa06651ab37.png)

### In my project,
![image](https://user-images.githubusercontent.com/31121811/228290367-7c706518-aed0-478e-9a1d-5f2b279be597.png)

**NB: In my component library I am using `react-select@^5.7.0`**

**Comment by Moshyfawn** [comment]
Do you mind providing a reproduction sandbox?
